spec/unit/form_spec.rb

```diff
@@ -116,6 +116,13 @@ def self.name
     end
 
     it 'raises error when attribute is in conflict with form interface' do
+      pending <<-PENDING.strip_heredoc
+      I've updated the builder to _skip_ already existing methods, rather
+      than raising a warning.
+      1) It's needed for allowing input to be called multiple times
+      2) If you've already defined a method, good on ya?
+      DISCUSS
+      PENDING
```

Comment:
I'd consider the chance of anyone coming from a rails background using attributes with those names in a rails app to be vanishingly small.

How about a check against a list of reserved methods to catch that case, then?